README.md

```diff
@@ -119,24 +122,6 @@ All the playbooks can be found inside the `examples/` folder.
     - role: ovirt.hosted_engine_setup
 ```
 
-## passwords.yml
-
-```yml
----
-# As an example this file is keep in plaintext, if you want to
-# encrypt this file, please execute following command:
-#
-# $ ansible-vault encrypt passwords.yml
-#
-# It will ask you for a password, which you must then pass to
-# ansible interactively when executing the playbook.
-#
-# $ ansible-playbook myplaybook.yml --ask-vault-pass
-#
-he_appliance_password: 123456
-he_admin_password: 123456
-```
-
```

Comment:
This is an example, please keep it as is.

Reply:
Done.